ffi/primitives/c/Makefile

```diff
@@ -8,7 +8,7 @@ dylib: primitives.o
 	gcc -dynamiclib -undefined suppress -flat_namespace primitives.o -o ../primitives.dylib
 
 dll: primitives.o
-	gcc -shared primitives.o -o ../primitives.dylib
+	gcc -shared primitives.o -o ../primitives.dll
```

Comment:
I think this file and the Makefile in the structs/ example need to have the `.dll` file  included in the `clean` step too, (like in the `hello_world` sample.)

@phanirithvij would you like to make that change?

Reply:
The [unit_test](https://travis-ci.org/dart-lang/samples/jobs/621903504) seems to have failed because of `package:test`'s version. Reverting to `0.39.1` might work.

Reply:
And I was trying to write windows tests for Travis on my fork in the `test` branch. https://github.com/phanirithvij/samples/pull/2.
I was able to test locally but on Travis, I couldn't get `nmake.exe` to run from dart `Process.run`.
I'd like some help setting it up.

Reply:
@johnpryan

Reply:
@johnpryan